data/service_updates/20240402.yml

```diff
@@ -0,0 +1,22 @@
+---
+title: Manage users in Manage school experience
+summary: |-
+  We've created a new feature that allows you to invite new school experience users directly from within Manage school experience.
+html_content: |-
+
+  <p>We’ve updated the Manage school experience service
+  so you can add new users directly on to your school’s account.</p>
+  <h3>New users</h3>
+  <p>When you add a new user to your school profile they can:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>advertise experience at your school</li>
+    <li>manage bookings and cancellations</li>
+    <li>add other people from your school</li>
+  </ul>
+  <h3>Adding a new user</h3>
+  <p>To add a new user:</p>
+  <ol class="govuk-list govuk-list--number">
+    <li>go to the school dashboard in your account</li>
+    <li>go to the Manage users section</li>
+    <li>add the new user</li>
```

Comment:
If you are changing the previous bullet to be `go to the Add new users section`, then you probably need to tweak this 

eg `enter the details of the new user`